AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant-base/add-participant-base.component.ts

```diff
@@ -398,4 +397,10 @@ export abstract class AddParticipantBaseDirective extends BookingBaseComponent i
             this.isInterpreter = false;
         }
     }
+
+    protected isNotAnObserver(participant): boolean {
```

Comment:
Just wondering if it would be better to reverse the logic and make this `isAnObserver`. It might make it simpler to re-use as you could call it with `this.isAnObserver() `rather than `!this.isNotAnObserver()`.

Reply:
Sure, that makes sense. Will update it.